### ISSUE TITLE
Q! New: New Version available

### ISSUE THREAD

**Issue by foreachthing** [Qalculate/qalculate-qt#13]
The dialog box, in Windows 10, has an issue with displaying a link to the download site:

![image](https://user-images.githubusercontent.com/10420187/131440289-e47e3fce-02b2-4e84-9add-c976c5672c13.png)

**Comment by JoeUX** [comment]
Note it's not just that the link formatting is broken. The new version pop-up is **incorrectly** saying that there's a new version available. I have 3.20.1 and it gives this pop-up every time I open the app, saying that 3.20.1 is available – that's the version I already have. It's not correctly comparing the installed version to the latest version available.

**Comment by foreachthing** [comment]
@JoeUX, you're right! Same here. 
I was just about to report that as well.

**Comment by JoeUX** [comment]
And to be clear, it only happens in the new QT client. 3.20.1 installs both the old and new clients. The old client is also identified as 3.20.1, but it doesn't exhibit this bug. It doesn't automatically generate the pop-up on opening the app, and if I manually Check for Updates in the Help menu, it says that there is no update available.

In the new QT client, if you dismiss the erroneous pop-up on app open, then manually Check for Updates in the menu, it will display the erroneous pop-up again, saying that 3.20.1 is available, even though that's what you have already.

**Comment by hanna-kn** [comment]
> The new version pop-up is incorrectly saying that there's a new version available.

The qalculate-qt binaries for version 3.20.1 were most likely incompletely recompiled (qmake does not automatically recompile files when defines in the project file have been updated). The issue is fixed in the later [64-bit "portable" zip package](https://github.com/Qalculate/qalculate-gtk/releases/download/v3.20.1/qalculate-3.20.1-x64.zip).

> I have 3.20.1 and it gives this pop-up every time I open the app,

Qalculate normally only automatically checks for new versions fortnightly, and should only inform the user once for each new version (which is true in my own tests). If this is not the case, it indicates that the application does not exit properly (the configuration file is not saved).

**Comment by JoeUX** [comment]
> Qalculate normally only automatically checks for new versions fortnightly, and should only inform the user once for each new version (which is true in my own tests). If this is not the case, it indicates that the application does not exit properly (the configuration file is not saved).

You're right. I must have made the wrong inference because I was opening it on multiple computers, so it seemed like every time. I tested it today and it doesn't pop up with the message. (I had to look up "fortnightly" though, since we don't use that word in the US. I thought it meant weekly, but apparently it's two weeks...)

I installed with the MSI installer, as I always do.
